lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -135,9 +135,9 @@
         required: yes
       rate:
         description:
-        - "Device read limit. Format: <number>[<unit>]"
+        - "Device read limit. Format: C(<number>[<unit>])"
```

Comment:
```suggestion
        - "Device read limit. Format: C(<number>[<unit>])."
```
Looks like the new sentence follows after.

Reply:
Good question. A period should only be at the end of a sentence, and `C(<number>[<unit>])` is not a sentence. In mathematics, I've seen the usage of `blabla: (some formula in a new line)` a lot where no dot was following the formula. (It's a different situation when there's no colon, but the formula is part of the sentence. And also if there's some more text directly after the sentence.) I would think of this here as a similar situation. But I'm not really sure :)

@acozine @samccann any opinions on this?

Reply:
Alternative proposal: rewrite as a proper sentence like `The format should be C(...).` Then it's part of a sentence and should be followed by a period. But that's less concise :)